cmd/utils.go

```diff
@@ -64,47 +64,56 @@ func checkLatestVersion() {
 		Owner:      "qlik-oss",
 		Repository: "corectl",
 	}
-
 	res, err := latest.Check(githubTag, version)
-
-	if err == nil && res.Outdated {
-
+	if err != nil {
+		internal.FatalError("Could not find latest version:", err)
+	}
+	if res.Outdated {
 		// Find absolute path of executable
 		executable, _ := os.Executable()
-
+		fmt.Println("--------------------------------------------------")
+		fmt.Printf("corectl version: %s, latest version is %s\n", version, res.Current)
+		switch runtime.GOOS {
+			//	case "darwin":
+			//		fmt.Println("To udpate to the latest version using brew just run:")
+			//		fmt.Print("\n  brew upgrade qlik-corectl\n\n")
+			//		fmt.Println("If you prefer using curl you can instead run:")
+		case "linux":
+			fmt.Println("To udpate to the latest version using snap just run:")
+			fmt.Print("\n  snap refresh qlik-corectl\n\n")
```

Comment:
why a mix of Println and Print with \n?

Reply:
The linter complains about \n in Println ^^